py_ballisticcalc/multiple_bc.py

```diff
@@ -53,8 +53,8 @@ def __init__(self, drag_table: Iterable[dict], diameter: Distance, weight: Weigh
     def _parse_mbc(self, mbc_table):
         table = []
         for p in mbc_table:
-            print(p['V'], Set.Units.velocity)
-            print(Set.Units.velocity(p['V']))
+            # print(p['V'], Set.Units.velocity)
+            # print(Set.Units.velocity(p['V']))
             v = Set.Units.velocity(p['V']) >> Velocity.MPS
```

Comment:
Why hardcoded MPS here?

Reply:
I don't know; I haven't worked on that module yet.  It looks like we just need to ensure that all MultiBCRow.v and MultiBC.speed_of_sound are in the same units.

I'm going to work through an example and add a unit test for this shortly.

Reply:
In this case it doesn't meter cause the velocity conversion is proportional and velocity applies to standart drag model as mach values, so I think double conversion not needed